examples/evs-csi-plugin/kubernetes/snapshot/pvc.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: pvc-snapshot-demo
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Gi
+  storageClassName: evs-sc
```

Comment:
pls add a black line

Reply:
done, has changed in branch "feat-documents-yaml"